epicsdbbuilder/const_array.py

```diff
@@ -0,0 +1,92 @@
+from decimal import Decimal
+from .recordbase import quote_string
+from .parameter import Parameter
+
+__all__ = ['ConstArray']
+
+
+class ConstArray:
+    """Constant Link Values. EPICS Base 7.0.4.1 and above.
```

Comment:
This documentation would be useful in `api.rst`, which at the moment is disappointingly bare.

Reply:
The intention was to favor IDE (with intellisense) over static text. Among others, a comment next to the code has higher chance for receiving updates rather than a static document. Yet, I recognize it depends on overall style of the documentation. Do you want me to duplicate the information in `api.rst` or move the information there leaving something much shorter here?